### PR TITLE
V2.13

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

**Comment by jasonnovichRunAI** [comment]
@omerbenedictrunai @yarongol I need a review of the new proposed changelog with all the versions and bug numbers that are listed here.

File: docs/home/changelog/hotfixes-2-13.md [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
+
+### Release content
+
+* <!-- RUN-14441 Add ability to set node affinity for Prometheus -->Added the ability to set node affinity for Prometheus.
+
+### Fixed issues
+
+| Internal ID | Description  |
+| ---------------------------- | ---- |
+| RUN-14472 | Fixed an issue where template updates were not being applieed to the workload. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-13.md:33]: Applieed

File: docs/home/changelog/hotfixes-2-13.md [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+| RUN-14434 | Fixed an issue where `runai_allocated_gpu_count_per_gpu` was multipled by seven. |
+| RUN-13956 | Fixed an issue where changing an exisitng template created a `Promise error` on existing job templates. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-13.md:35]: ....when changing an existing template in the Jobs screen...

File: docs/home/changelog/hotfixes-2-13.md [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
+| RUN-13825 | Fixed an issue where GPU sharing configmaps are leaked. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-13.md:36]: Very frightening and not relevant, plase remove this bug

File: docs/home/changelog/hotfixes-2-13.md [hunk 4]
```diff
@@ -32,12 +54,43 @@ The following is a list of the known and fixed issues for Run:ai V2.13.
...
+* <!--RUN-12349	Enable quota management by default -->Quota management is now enabled by default.
 
 | Internal ID | Description  |
 | ---------------------------- | ---- |
 | RUN-12923 | Fixed an issue in upgrading due to a misconfigured image for airgapped systems in 2.13.19. The helm chart contained an error, and the image is not used even though it is packaged as part of the tar. |
 | RUN-12928, </br>RUN-12968 | Fixed an issue in upgrading Prometheus due to a misconfigured image for airgapped systems in 2.13.19. The helm chart contained an error, and the image is not used even though it is packaged as part of the tar. |
+| RUN-12751 | Fixed an issue when upgrading from 2.9 to 2.13 results with a missing engine-config file. |
+| RUN-12717 | Fixed an issue where the user that is logged in as researcher manager can't see the clusters. |
+| RUN-12642 | Fixed an issue where Assets-sync could not restart due to failing to get token from control plane. |
+| RUN-12191 | Fixed an issue where there was a Time out while waiting for the `runai_allocated_gpu_count_per_project` metric to return values. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-13.md:71]: timeout - one word no capital 'T'

File: docs/home/changelog/hotfixes-2-13.md [hunk 5]
```diff
@@ -32,12 +54,43 @@ The following is a list of the known and fixed issues for Run:ai V2.13.
...
+### Fixed issues
+
+| Internal ID | Description  |
+|-------------- | ------------------- |
+| RUN-12650 | Fixed an issue that used an incorrect metric in analytics GPU ALLOCATION PER NODE panel. Now the correct allocation is in percentage. |
+| RUN-12602 | Fixed an issue in `runaiconfig` where `WorkloadServices` have any memory requests/limits and cpu requests/limits and is overwritten with a default. |
+| RUN-12587 | Fixed an issue where the scheduler enters an endless loop running elastic jobs. |
+| RUN-12585 | Fixed an issue where the workload-controller creates a delay in running jobs. |
+| RUN-12031 | Fixed an issue when upgrading from 2.9 to 2.13 where the Scheduler pod fails to upgrade due to the change of owner. |
+| RUN-11091  | Fixed an issue when `departments FF` is off you are not able to schedule non-preemable jobs. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-13.md:93]: department FF is not customer facing. Use th following customer facing option:
"Enable Departments" option under Settings

File: docs/home/changelog/hotfixes-2-13.md (hunk 2 above)
Comment (yarongol) [reply docs/home/changelog/hotfixes-2-13.md:35]: I think it should be "multiplied"

File: docs/home/changelog/hotfixes-2-13.md (hunk 3 above)
Comment (yarongol) [reply docs/home/changelog/hotfixes-2-13.md:36]: I don't know what "GPU sharing" means in this context. If swap - not interesting. If fractions, we can phrase that more gently as "when deleting a job that is allocated a fraction of a GPU, an associated configmap is not deleted"

File: docs/home/changelog/hotfixes-2-13.md [hunk 6]
```diff
@@ -32,12 +54,43 @@ The following is a list of the known and fixed issues for Run:ai V2.13.
...
+* <!--RUN-12420 Show only logged-in user's jobs -->Added functionality that displays Jobs that only belong to the user that is logged in.
+* <!-- RUN-12405 Refine out of the box alerts to run:ai rules --> Added and refined alerts to the state of Run:ai components, schedule latency, and warnings for out of memory on Jobs.
+* <!-- RUN-11711 Pod Security Admission issues in control plane -->Added the ability to work with restricted PSA policy.
+
+### Fixed issues
+
+| Internal ID | Description  |
+|-------------- | ------------------- |
+| RUN-12650 | Fixed an issue that used an incorrect metric in analytics GPU ALLOCATION PER NODE panel. Now the correct allocation is in percentage. |
+| RUN-12602 | Fixed an issue in `runaiconfig` where `WorkloadServices` have any memory requests/limits and cpu requests/limits and is overwritten with a default. |
```
Comment (yarongol) [inline docs/home/changelog/hotfixes-2-13.md:89]: I don't understand the statement as written

File: docs/home/changelog/hotfixes-2-13.md [hunk 7]
```diff
@@ -32,12 +54,43 @@ The following is a list of the known and fixed issues for Run:ai V2.13.
...
+| RUN-12587 | Fixed an issue where the scheduler enters an endless loop running elastic jobs. |
```
Comment (yarongol) [inline docs/home/changelog/hotfixes-2-13.md:90]: What is an "elastic job"?

File: docs/home/changelog/hotfixes-2-13.md (hunk 6 above)
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-13.md:89]: From what this bug looks like, it seems that there are settings in the `runaiconfig` in the `workloadservices` section that were overwritten with default settings. Meaning the confgig was ignored, superseded, or not working (essence of the bug). I can rephrase to:

```suggestion
| RUN-12602 | Fixed an issue in `runaiconfig` where the `WorkloadServices` spec has memory requests/limits and cpu requests/limits and gets overwritten with the system default. |
```

File: docs/home/changelog/hotfixes-2-13.md (hunk 7 above)
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-13.md:90]: @yarongol I am going to assume that it means this https://kubernetes.io/docs/concepts/workloads/controllers/job/#elastic-indexed-jobs. Correct me if I didn't get it, but it seems that an elastic job is a form of autoscaling jobs. So maybe the scheduler has an issue with autoscaling and the above API reference?

File: docs/home/changelog/hotfixes-2-13.md (hunk 5 above)
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-13.md:93]: @omerbenedictrunai
```suggestion
| RUN-11091  | Fixed an issue where the *Departments* feature is disabled, you are not able to schedule non-preemable jobs. |
```

File: docs/home/changelog/hotfixes-2-13.md (hunk 3 above)
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-13.md:36]: @omerbenedictrunai 
```suggestion
| RUN-13825 | Fixed an issue when deleting a job that is allocated a fraction of a GPU, an associated configmap is not deleted. |
```

File: docs/home/changelog/hotfixes-2-13.md [hunk 8]
```diff
@@ -32,12 +54,43 @@ The following is a list of the known and fixed issues for Run:ai V2.13.
...
+## Version 2.13.20 - September 28, 2023
+
+### Release content
+
+* <!-- RUN-12537 hide credentials from the UI when cluster version is not compatible -->Added the prevention of selecting tenant or department scopes for credentials, and the prevention of selecting s3, PVC, and Git data sources if the cluster version does not support these.
+* <!--RUN-12349	Enable quota management by default -->Quota management is now enabled by default.
 
 | Internal ID | Description  |
 | ---------------------------- | ---- |
 | RUN-12923 | Fixed an issue in upgrading due to a misconfigured image for airgapped systems in 2.13.19. The helm chart contained an error, and the image is not used even though it is packaged as part of the tar. |
```
Comment (yarongol) [inline docs/home/changelog/hotfixes-2-13.md:66]: image --> docker image

Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-13.md:66]:
```suggestion
| RUN-12923 | Fixed an issue in upgrading due to a misconfigured Docker image for airgapped systems in 2.13.19. The helm chart contained an error, and the image is not used even though it is packaged as part of the tar. |
```

File: docs/home/changelog/hotfixes-2-13.md (hunk 7 above)
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-13.md:90]: @omerbenedictrunai 
```suggestion
| RUN-12587 | Fixed an issue where elastic scheduling of jobs enters an endless loop. |
```

**Review by yarongol — APPROVED** [review]

File: docs/home/changelog/hotfixes-2-13.md [hunk 9]
```diff
@@ -32,12 +54,43 @@ The following is a list of the known and fixed issues for Run:ai V2.13.
...
 | RUN-12928, </br>RUN-12968 | Fixed an issue in upgrading Prometheus due to a misconfigured image for airgapped systems in 2.13.19. The helm chart contained an error, and the image is not used even though it is packaged as part of the tar. |
+| RUN-12751 | Fixed an issue when upgrading from 2.9 to 2.13 results with a missing engine-config file. |
+| RUN-12717 | Fixed an issue where the user that is logged in as researcher manager can't see the clusters. |
+| RUN-12642 | Fixed an issue where Assets-sync could not restart due to failing to get token from control plane. |
```
[the remark this answers is not in the document]
Comment (yarongol) [reply docs/home/changelog/hotfixes-2-13.md:70]: See comment @jasonnovichRunAI

[the remark this answers is not in the document]
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-13.md:70]:
```suggestion
| RUN-12642 | Fixed an issue where assets-sync could not restart due to failing to get token from control plane. |
```

**Pull request merged**